src/Apps/Order/Utils/orderUtils.ts

```diff
@@ -10,27 +10,12 @@ const initialPaymentMethods: CommercePaymentMethodEnum[] = [
   "WIRE_TRANSFER",
 ]
 
-export const isPaymentSet = (
```

Comment:
Retire the entire utility to use the field returned from Exchange. This refactor is in [a63a36b](https://github.com/artsy/force/pull/14709/commits/a63a36b863f0386722729983ae8c570679cb3209).